Verdaccio 2.4 returns wrong package.json

#### My reason:
I setup a verdaccio 2.4 server yesterday and found many projects failed to build after reinstall dependencies.After several hours digging,i found verdaccio returns wrong package.json file which remove some info from original package.json, e.g, "main" field. So it cause nodejs to throw many "can not find module' errors. Finally i revert to verdaccio 2.2, and all works fine.

#### Steps to reproduce:

<!-- 
(1) Setup a clean verdaccio 2.4 server;
(2) install @angular/cli and create a new angular project;
(3) use "ng build" to build project would throw error.

-->
####  App Version:	 2.4

One things I have noticed that you have to use `npm install -g @angular/cli` but in the conversation @jackjoy mentioned `npm install -g angular/cli`. 

`npm install -g angular/cli`  produces following error:

```
➜  verdaccio git:(master) ✗ npm install -g angular/cli
npm ERR! git clone --template=/Users/ayush/.npm/_git-remotes/_templates --mirror git@github.com:angular/cli.git /Users/ayush/.npm/_git-remotes/git-github-com-angular-cli-git-05516032: Cloning into bare repository '/Users/ayush/.npm/_git-remotes/git-github-com-angular-cli-git-05516032'...
npm ERR! git clone --template=/Users/ayush/.npm/_git-remotes/_templates --mirror git@github.com:angular/cli.git /Users/ayush/.npm/_git-remotes/git-github-com-angular-cli-git-05516032: ERROR: Repository not found.
npm ERR! git clone --template=/Users/ayush/.npm/_git-remotes/_templates --mirror git@github.com:angular/cli.git /Users/ayush/.npm/_git-remotes/git-github-com-angular-cli-git-05516032: fatal: Could not read from remote repository.
npm ERR! git clone --template=/Users/ayush/.npm/_git-remotes/_templates --mirror git@github.com:angular/cli.git /Users/ayush/.npm/_git-remotes/git-github-com-angular-cli-git-05516032:
npm ERR! git clone --template=/Users/ayush/.npm/_git-remotes/_templates --mirror git@github.com:angular/cli.git /Users/ayush/.npm/_git-remotes/git-github-com-angular-cli-git-05516032: Please make sure you have the correct access rights
npm ERR! git clone --template=/Users/ayush/.npm/_git-remotes/_templates --mirror git@github.com:angular/cli.git /Users/ayush/.npm/_git-remotes/git-github-com-angular-cli-git-05516032: and the repository exists.
npm ERR! Darwin 16.7.0
npm ERR! argv "/Users/ayush/.nvm/versions/node/v6.10.1/bin/node" "/Users/ayush/.nvm/versions/node/v6.10.1/bin/npm" "install" "-g" "angular/cli"
npm ERR! node v6.10.1
npm ERR! npm  v3.10.10
npm ERR! code 128

npm ERR! Command failed: git clone --template=/Users/ayush/.npm/_git-remotes/_templates --mirror git@github.com:angular/cli.git /Users/ayush/.npm/_git-remotes/git-github-com-angular-cli-git-05516032
npm ERR! Cloning into bare repository '/Users/ayush/.npm/_git-remotes/git-github-com-angular-cli-git-05516032'...
npm ERR! ERROR: Repository not found.
npm ERR! fatal: Could not read from remote repository.
npm ERR!
npm ERR! Please make sure you have the correct access rights
npm ERR! and the repository exists.
npm ERR!
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/ayush/xyz/verdaccio/npm-debug.log
```

--- a/src/lib/storage/up-storage.js
+++ b/src/lib/storage/up-storage.js
@@ -17,7 +17,7 @@ const encode = function(thing) {
 
 const jsonContentType = 'application/json';
 
-const contenTypeAccept = `${jsonContentType}; q=0.8, */*`;
+const contenTypeAccept = `${jsonContentType};`;
 
 /**
  * Just a helper (`config[key] || default` doesn't work because of zeroes)

